lib/reverse_coverage/formatters/html/formatter.rb

```diff
@@ -58,21 +58,53 @@ def formatted_source_file(source_file)
         end
 
         def readfile(source_file)
-          File.open(filename(source_file), "rb", &:readlines)
+          load_source(filename(source_file))
+        end
+
+        def load_source(file_name)
+          lines = []
+          # The default encoding is UTF-8
+          File.open(file_name, "rb:UTF-8") do |file|
+            line = file.gets
+
+            # Check for shbang
+            if /\A#!/.match?(line)
+              lines << line
+              line = file.gets
+            end
+            return lines unless line
+
+            check_magic_comment(file, line)
+            lines.concat([line], file.readlines)
+          end
+
+          lines
+        end
+
+        def check_magic_comment(file, line)
+          # Check for encoding magic comment
+          # Encoding magic comment must be placed at first line except for shbang
+          if (match = /\A#\s*(?:-\*-)?\s*(?:en)?coding:\s*(\S+)\s*(?:-\*-)?\s*\z/.match(line))
+            file.set_encoding(match[1], "UTF-8")
+          end
         end
 
         def grouped(files)
           grouped = {}
           grouped_files = []
 
+          # TODO Someday / Maybe. Make this list configurable
           groups = {
             'Controllers' => %r{/app/controllers},
             'Channels' => %r{/app/channels},
             'Models' => %r{/app/models},
             'Mailers' => %r{/app/mailers},
             'Helpers' => %r{/app/helpers},
             'Jobs' => %r{/app/jobs|/app/workers},
-            'Libraries' => %r{/lib/}
+            'Libraries' => %r{/lib/},
+            'Interactors' => %r{/app/interactors},
+            'Services' => %r{/app/services},
+            'Validators' => %r{/app/validators},
```

Comment:
should probably removes these 3 last lines. Not standard

Reply:
Yes, maybe you can add a configuration in order to customize the Categories

Reply:
It's been 2 years. I won't work on it. You can remove these lines with a new commit

Reply:
I cannot edit your branch. I would prefer it if you can remove that commit and force-push the change.